lib/StatusIndicator/styles.scss

```diff
@@ -2,30 +2,55 @@
    Config
    ========================================================================== */
 
-$status-indicator-size: 1em !default;
 $status-indicator-label-color--text: rgb(111, 121, 129) !default;
 
 /* ==========================================================================
    Styles
    ========================================================================== */
 
 .status-indicator {
-  display: inline-block;
-  vertical-align: middle;
   color: $status-indicator-label-color--text;
+  .due-date {
+    color: #8492a6;
+    font-size: 12px;
```

Comment:
Always try to use font-size variables from `settings/_typography.scss` when defining a font size. E.g. if you want this to be smaller then `$typo-size-slight` or `$typo-size-small` will suit your needs.